exporter/opentelemetry-exporter-datadog/src/opentelemetry/exporter/datadog/spanprocessor.py

```diff
@@ -81,6 +82,24 @@ def __init__(
         self.schedule_delay_millis = schedule_delay_millis
         self.done = False
         self.worker_thread.start()
+        # Only available in *nix since py37.
+        if hasattr(os, "register_at_fork"):
+            os.register_at_fork(
+                after_in_child=self._at_fork_reinit
+            )  # pylint: disable=protected-access
+
+    def _at_fork_reinit(self):
+        self.condition = threading.Condition(threading.Lock())
+        self.flush_condition = threading.Condition(threading.Lock())
+        self.traces_lock = threading.Lock()
+        self.check_traces_queue.clear()
+        self.worker_thread = self._create_worker_thread()
+        self.worker_thread.start()
```

Comment:
I really wasn't too sure here what I should/shouldn't re-initialize. I figured I would initialize all of it just in case